examples/jetstream/partitions/meta.yaml

```diff
@@ -0,0 +1,26 @@
+title: Subject-Mapped Partitions
+description: |-
+  A stream provides total order over the set of messages
+  it stores. If configured with replicas, each stream has a
+  dedicated Raft group to ensure every message published to
+  the stream is replicated to the majority of servers having a
+  replica. On successful a publish, every replica of the stream
+  is guaranteed to have the same order of messages. Similiarly,
+  consumers on that stream will always receive the messages in
```

Comment:
Notice that only 'Ordered' consumer guarantees ordered delivery.

Regular consumer may deliver out of order (although they don't in most cases).

Reply:
Understood, I indicated the happy path. I chose not to talk about re-deliveries and concurrent subs bound to a stream. I can clarify to avoid claiming any guarantee if you prefer.

Reply:
I agree this example may not be the best place to have that kind of digression. Just wanted to highlight, but you know best what and what doesn't belong here.